rtengine/canon_cr3_decoder.cc

```diff
@@ -450,8 +468,8 @@ int DCraw::parseCR3(unsigned long long oAtomList,
       }
 #define current_track RT_canon_CR3_data.crx_header[nTrack]
 
-      ImageWidth = get2();
-      ImageHeight = get2();
+//      ImageWidth = get2();
```

Comment:
Couldn't it be possible the get2() are still needed because the change the fileposition?

Reply:
You're right, though it still decoded...